terror/codes.go

```diff
@@ -11,17 +11,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package errors
+package terror
```

Comment:
How about removing codes.go content to terror.go?
Seems only one source file is OK?